vscode-dotnet-sdk-extension/src/extension.ts

```diff
@@ -157,19 +157,24 @@ export function activate(context: vscode.ExtensionContext, extensionContext?: IE
 }
 
 function setPathEnvVar(pathAddition: string, displayWorker: IWindowDisplayWorker) {
-    if (process.env.PATH && process.env.PATH.includes(pathAddition)) {
-        // No need to add to PATH again
-        return;
-    }
-
     let pathCommand: string;
     if (os.platform() === 'win32') {
+        if (process.env.PATH && process.env.PATH.includes(pathAddition)) {
+            // No need to add to PATH again
+            return;
+        }
         pathCommand = `for /F "skip=2 tokens=1,2*" %A in ('%SystemRoot%\\System32\\reg.exe query "HKCU\\Environment" /v "Path" 2^>nul') do ` +
-                      `(%SystemRoot%\\System32\\reg.exe ADD "HKCU\\Environment" /v Path /t REG_SZ /f /d "${pathAddition};%C")`;
+            `(%SystemRoot%\\System32\\reg.exe ADD "HKCU\\Environment" /v Path /t REG_SZ /f /d "${pathAddition};%C")`;
+
     } else {
-        // Adding to the user path is only supported on windows right now
-        return;
+        const profileFile = os.platform() === 'darwin' ? path.join(os.homedir(), '.zshrc') : path.join(os.homedir(), '.profile');
```

Comment:
Is there a prompt? rc file is very "private". We should not edit that write without explicit permission

Reply:
As discussed offline, there isn't a prompt but we've decided that's fine since the extension will only be used by the education bundle which is targeted for students/ new developers who will likely not understand rc files.